### PR TITLE
[bulk] validation: validation fixes JM 5/13

### PR DESCRIPTION
**Pull request by jmart1428**
This PR fixes 80 issues.

### CODE REVIEW

File: aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For this tutorial, you'll add mobile features to the simple conference-listing a
 
 You can use Internet Explorer version 9 or 10, FireFox or Chrome to develop your mobile application by setting the [user agent string](http://www.howtogeek.com/113439/how-to-change-your-browsers-user-agent-without-installing-any-extensions/). The following image shows the completed tutorial using Internet Explorer emulating an iPhone. You can use the Internet Explorer F-12 developer tools and the [Fiddler tool](http://www.fiddler2.com/fiddler2/) to help debug your application.
 
-![](aspnet-mvc-4-mobile-features/_static/image3.png)
+![Screenshot that shows the All tags page on a Windows 7 phone display.](aspnet-mvc-4-mobile-features/_static/image3.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md:44]: 100 characters recommended for alt-text. You're halfway there!

File: aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ Install the *jQuery.Mobile.MVC* NuGet package by doing this:
 
 The following image shows the files added and changed to the MvcMobile project by the NuGet jQuery.Mobile.MVC package. Files which are added have [add] appended after the file name. The image does not show the GIF and PNG files added to the *Content\images* folder.
 
-![](aspnet-mvc-4-mobile-features/_static/image21.png)
+![Screenshot that shows the Microsoft Visual Web Developer Window. Library Package Manager is selected in the Tools drop down menu. Package Manager Console is selected in the sub menu.](aspnet-mvc-4-mobile-features/_static/image21.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md:187]: To get this down to 150 characters, I'd just remove "the" from last sentence.

File: aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md [hunk 3]
```diff
@@ -289,7 +289,7 @@ The following image shows the search filter box at the top of the page that resu
 
 As you type each letter in the search box, jQuery Mobile filters the displayed list as shown in the image below.
 
-[![ps_data_filter_SC](aspnet-mvc-4-mobile-features/_static/image39.png)](aspnet-mvc-4-mobile-features/_static/image38.png)
+[![Screenshot that shows the All Speakers page in mobile view.](aspnet-mvc-4-mobile-features/_static/image39.png)](aspnet-mvc-4-mobile-features/_static/image38.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md:292]: 100 characters.

File: aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md [hunk 4]
```diff
@@ -331,7 +331,7 @@ In this section, you'll create a mobile-specific view of sessions. The changes w
 
 In the mobile browser, tap the **Speaker** button, then enter `Sc` in the search box.
 
-[![ps_data_filter_SC](aspnet-mvc-4-mobile-features/_static/image47.png)](aspnet-mvc-4-mobile-features/_static/image46.png)
+[![Screenshot that shows the All Speakers page in mobile view with the letters S c entered into the search.](aspnet-mvc-4-mobile-features/_static/image47.png)](aspnet-mvc-4-mobile-features/_static/image46.png)
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/aspnet-mvc-4-mobile-features.md:334]: 100 characters.

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/accessing-your-models-data-from-a-controller.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/accessing-your-models-data-from-a-controller
 title: "Accessing your Model's Data from a Controller (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to create a movie, examine generated code, pass strongly typed data or objects to a view template, and work with SQL server compact."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/accessing-your-models-data-from-a-controller.md:5]: To keep this description within 150 characters, I'd go with "Learn how to create a movie, examine generated code, pass strongly typed data or objects to a view template, and work with SQL server compact."

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-model.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-model
 title: "Adding a Model (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "Note: An updated version of this tutorial is available here that uses ASP.NET MVC 5 and Visual Studio 2013. It's more secure, much simpler to follow and demo..."
+description: This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to add a model and model class and also work with SQL server compact to create a connection string."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-model.md:5]: This gets it down to 144 characters: "Learn to build an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. Work with SQL server compact to create a connection string."

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-new-field.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-new-field
 title: "Adding a New Field to the Movie Model and Table (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to add a rating property to the movie model, manage model and database schema differences, and automatically re-create the database on model changes."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-new-field.md:5]: "Learn how to add a rating property to the movie model, manage model and database schema differences, and automatically re-create the database on model changes."

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-view.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-view
 title: "Adding a View (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to change the Views and Layout pages and pass data from the Controller to the View."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-view.md:5]: "Learn how to change the Views and Layout pages and pass data from the Controller to the View using Microsoft Visual Web Developer 2010 Express Service Pack 1."

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/improving-the-details-and-delete-methods.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/improving-the-details-and-delete-methods
 title: "Improving the Details and Delete Methods (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to make improvements to the automatically generated Details and Delete methods. With just a few small bits of code, you can easily enhance the application."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/improving-the-details-and-delete-methods.md:5]: Edit this to remove Service Pack boilerplate.

File: aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/adding-a-model.md [hunk 10]
```diff
@@ -1,15 +1,15 @@
 ---
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc4/adding-a-model
-title: "Adding a Model | Microsoft Docs"
+title: "Adding a Model 2012 | Microsoft Docs"
 author: Rick-Anderson
-description: "Note: An updated version of this tutorial is available here that uses ASP.NET MVC 5 and Visual Studio 2013. It's more secure, much simpler to follow and demo..."
+description: "In this section you'll add some classes for managing movies in a database. These classes will be the model part of the ASP.NET MVC app."
```
Comment (kturetz) [inline aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/adding-a-model.md:5]: In descriptions, we eliminate introductory phrases such as "In this tutorial" and get right to the point. "Add some classes for managing movies...."

**Comment by jmart1428** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**